asapdiscovery-data/asapdiscovery/data/operators/selectors/mcs_selector.py

```diff
@@ -208,7 +212,11 @@ def _select(
 
             pattern_query = oechem.OEQMol(ligand.to_oemol())
             pattern_query.BuildExpressions(atomexpr, bondexpr)
-            mcss = oechem.OEMCSSearch(pattern_query)
+            if self.approximate:
+                mcs_stype = oechem.OEMCSType_Approximate
```

Comment:
should log this as `warning`, possibly referring to docs: https://docs.eyesopen.com/toolkits/python/oechemtk/patternmatch.html

Reply:
Good idea.